src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -94,91 +127,108 @@
+                                                         {:currency fiat-currency
+                                                          :balance  fee-in-native-token
+                                                          :token    native-token}))
+            currency-symbol                           (rf/sub [:profile/currency-symbol])
+            fee-formatted                             (when fee-in-fiat
+                                                        (utils/get-standard-fiat-format
+                                                         fee-in-crypto-formatted
+                                                         currency-symbol
+                                                         fee-in-fiat))
+            show-select-asset-sheet                   #(rf/dispatch
+                                                        [:show-bottom-sheet
+                                                         {:content (fn []
+                                                                     [select-asset-bottom-sheet
+                                                                      clear-input!])}])
+            sender-network-values                     (rf/sub
+                                                       [:wallet/wallet-send-sender-network-values])
+            receiver-network-values                   (rf/sub
+                                                       [:wallet/wallet-send-receiver-network-values])
+            token-not-supported-in-receiver-networks? (every? #(= (:type %) :not-available)
+                                                              (filter #(not= (:type %) :add)
+                                                                      receiver-network-values))
+            suggested-routes                          (rf/sub [:wallet/wallet-send-suggested-routes])
+            routes                                    (when suggested-routes
+                                                        (or (:best suggested-routes) []))
+            no-routes-found?                          (and
+                                                       (every-network-value-is-zero?
+                                                        sender-network-values)
+                                                       (not (nil? routes))
+                                                       (not loading-routes?)
+                                                       (not token-not-supported-in-receiver-networks?))
+            receiver-networks                         (rf/sub [:wallet/wallet-send-receiver-networks])
+            receiver-preferred-networks               (rf/sub
+                                                       [:wallet/wallet-send-receiver-preferred-networks])
+            sending-to-unpreferred-networks?          (not= (set receiver-networks)
+                                                            (set receiver-preferred-networks))]
```

Comment:
imo having such a long list of definitions that way can be more prone to introducing bugs and harder to debug, what do you guys think? @J-Son89 @briansztamfater

Reply:
You mean if variable definitions or this particular variable?

If you mean the list of variables then yes I agree it would be great o further break down these components and isolate further some of the subs here if possible

Reply:
> If you mean the list of variables then yes

Yes that's what I mean.